### ISSUE TITLE
vimwiki help not working

### ISSUE THREAD

**Issue by DaveLG** [vimwiki/vimwiki#243]
Well I have vimwiki installed on a Windows 10 machine with gvim. It seems to work as I can see vimwiki in the top menu as well as look at my old wiki's that I had in drop box with MacOS and MacVim.

I am missing vimwiki help though. I have tried these two commands in gvim but neither worked:

:helptags "C:\Program Files (x86)\Vim\vimfiles\bundle\vimwiki\doc\vimwiki.txt"
E471: Arguement required

:helptags "C:\Program Files (x86)\Vim\vimfiles\bundle\vimwiki\doc"
E471: Arguement required

If I run :Helptags nothing seems to happen. The cursor just returns to the edit window

Any ideas?

**Comment by EinfachToll** [comment]
Maybe try `:helptags ALL`, which builds, according to the documentation, the tags for all paths in 'runtimepath'.

> :helptags "C:\Program Files (x86)\Vim\vimfiles\bundle\vimwiki\doc"
E471: Arguement required

I don't really understand why the error message pops up. Maybe Vim doesn't expect the path to be enclosed in ""? Try `:helptags C:\\Program\ Files\ (x86)\\Vim\\vimfiles\\bundle\\vimwiki\\doc`.

> If I run :Helptags nothing seems to happen. The cursor just returns to the edit window

It does so when it is successful, too.

**Comment by DaveLG** [comment]
If I try  :helptags C:\\Program\ Files\ (x86)\\Vim\\vimfiles\\bundle\\vimwiki\\doc  without the quotes I get this

`E153 Cannot open C:\\Program\ Files\ (x86)\\Vim\\vimfiles\\bundle\\vimwiki\\doc\tags for writing`
     Note the tag appended to the end.

I was able to set up gvim with the Windows 10 Ubuntu bash new feature.    :h vimwiki works there.

I also just noticed that there is no help available for Tabular which is installed and works. I use Pathogen.

So maybe the fault is not just particular to vimwiki. I did a 
:set runtimepath? 
and get a list of Windows directories ( I haven't figured out how to copy the command's output  or send the output to a file or  I would post it here. (Chances of me correctly typing it is near zero!) 

I am thinking maybe a runtime path is bad. Also maybe pathogen is missing something.

Thanks for the ideas.

Dave

**Comment by EinfachToll** [comment]
The `:helptags` command tries to write to this particular tags file, but Vim can't. Do you have writing access to the directory, i.e. can you manually create a file in "C:\\Program\ Files\ (x86)\\Vim\\vimfiles\\bundle\\vimwiki\\doc"?

BTW, did you just mistype it or is there just one backslash between `doc` and `tags` in the error message?

> I haven't figured out how to copy the command's output or send the output to a file or I would post it here. 

Select the output using the mouse and press Ctrl+y (works at least under Linux).

**Comment by DaveLG** [comment]
I tried using just Notepad to write a file to the C:\Program\ Files\ (x86)\Vim\vimfiles\bundle\vimwiki\doc folder and was denied permission.  I couldn't just write a file via gvim to the location either. So that led me to think the permission on the folder was wrong. 

That was it although it wasn't straight forward. I was use to just right clicking on a folder and changing permissions but that didn't work even though I checked what I thought were the appropriate check boxes. One has to enter an Advanced Security menu and add some sort of object for my user name. 

I can't say I recall exactly what I did but it works for vimwiki help files plus all others.

Maybe I should have had the setup folders and plug in stuff located somewhere outside of the Windows Program Files (x86) folder tree.

As I get familiar with this I may make modifications etc.

BTW the error message I initially got wasn't 153, it was either 154 or 155. 

Thanks very much for your assistance. I've been working on and off for a day on this. You set me on the correct path!!
